Stepic/Legacy/Model/Entities/Course/Course+CoreDataProperties.swift

```diff
@@ -318,6 +321,24 @@ extension Course {
         }
     }
 
+    var displayPriceTierPrice: String? {
+        get {
+            self.managedDisplayPriceTierPrice
```

Comment:
"self." is mostly just a syntax sugar. why are you writing it?

Reply:
This is a rule from our Swift Style Guide.
See for example https://realm.github.io/SwiftLint/explicit_self.html.